enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx

```diff
@@ -34,7 +34,7 @@ import { SdkFontsGlobalStyles } from "../private/SdkGlobalFontsStyles";
 import { PortalContainer } from "../private/SdkPortalContainer";
 import { SdkUsageProblemDisplay } from "../private/SdkUsageProblem";
 
-import "metabase/css/index.module.css";
+import "metabase/embedding-sdk/css/index.module.css";
 import "metabase/css/vendor.css";
```

Comment:
I completely don't like that we have to import `mantine`, `main` styles and `vendor` separately.

But if we put everything into a single `index.module.css` file and define layers there (that looks much better), our css-loader that transforms css modules also renames class names of `mantine` and `vendor` files, that we must not rename. And though we have the:
`auto: (filename) =>
      !filename.includes("node_modules") && !filename.includes("vendor.css"),`, the `auto` function is called for each `css module` file referenced from `js` file, but is not called for `@import` statements inside the `css` file